packages/loopring_v3/contracts/impl/ExchangeV3.sol

```diff
@@ -91,31 +101,34 @@ contract ExchangeV3 is IExchangeV3
         );
     }
 
+    function setDepositContract(address _depositContract)
+        external
+        onlyOwner
+    {
+        // Only used for initialization
```

Comment:
Added just in case some kind of logic depends on this. But the exchange is pretty much useless until a valid deposit contract is set because no deposits are possible without a valid deposit contract.

Reply:
Added just in case some kind of logic depends on this. But the exchange is pretty much useless until a valid deposit contract is set because no deposits are possible without a valid deposit contract.